app/src/androidTest/java/edu/nextstep/camp/calculator/MainActivityTest.kt

```diff
@@ -0,0 +1,93 @@
+package edu.nextstep.camp.calculator
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import org.junit.Rule
+import org.junit.Test
+
+class MainActivityTest {
+    @get:Rule
+    var activityScenarioRule = ActivityScenarioRule(MainActivity::class.java)
+
+    //사용자가 피연산자 0 ~ 9 버튼을 누르면 화면에 해당 숫자가 화면에 보여야 한다.
+
+    @Test
+    fun `버튼_0을_누르면_화면에_0이_보여야한다`() {
+        //when : 0 click
+        onView(ViewMatchers.withId(R.id.button0)).perform(click())
+        //then : display 0
+        onView(ViewMatchers.withId(R.id.tv_result)).check(ViewAssertions.matches(withText("0")))
+    }
+
+    @Test
+    fun `버튼_1을_누르면_화면에_1이_보여야한다`() {
+        //when : 0 click
+        onView(ViewMatchers.withId(R.id.button1)).perform(click())
```

Comment:
withId도 static import 형태로 사용하시면,
코드의 가독성이 더욱 올라갈것 같습니다. 🙏
```suggestion
        onView(withId(R.id.button1)).perform(click())
```